src/client/src/app/pages/page-games/page-games.component.html

```diff
@@ -1 +1,3 @@
 <p>page-games works!</p>
+
```

Comment:
sort of confused as to where to add this comment .. but the ts page for this html should use the get all games selector to get from the database but that is never done. the ts page is clear

Reply:
yeah.. I cleared it out because I am getting an error where none of data are showing though they are showing in redux. So I thought I will just create a separate issue for that.

Reply:
ok for sure, that sounds like a plan.. plus you already have all the leg work in place.. shouldnt be too hard of an issue